contracts/src/interfaces/IPayer.sol

```diff
@@ -2,22 +2,20 @@
 pragma solidity 0.8.28;
```

Comment:
Actually, can you remove IPayer completely from this PR? I'll be handling these changing in its own PR.